embedly/embedly/api/views.py

```diff
@@ -43,3 +43,14 @@ def extract_urls():
         status=200,
         mimetype='application/json',
     )
+
+
+@blueprint.route('/v2/extract', methods=['POST'])
+def extract_urls_v2():
+    urls = json.loads(request.values.get('urls', '[]'))
```

Comment:
on the same topic, flask's request supports a `json` property.

The `request.json` property will only return if the request's mime type is set to `application/json`, which we should do anyway.

We can take advantage of that, but not a big deal either